_acs-aem-commons/features/ms-office-asset-selector/index.md

```diff
@@ -31,30 +56,53 @@ a method for distributing the add-in to users easily.
 
 ### AEM Configuration
 
-Cerate a new page that will act as the end-point for the MS Office Add-in interact with.
+Install the above content package or follow the steps below after including the full ACS Commons package.
+
+Create a new page that will act as the end-point for the MS Office Add-in interact with.
 
-Use ACS Commons' MS Office Asset Selector static template, which is configured to be available under the path `/content/asset-selectors`
+Use ACS Commons' MS Office Asset Selector static template, which is configured to be available under the
+path `/content/asset-selectors`.
 
 ![AEM Page](../ms-office-asset-selector/images/aem-page.jpeg)
 
-For the mot part this is all that is neccessary to begin using this add-in. 
+For the most part, this is all that is necessary to begin using this add-in.
 However, you may want to review the information about [managing access](#managing_access)
 for more information on how access to content can be controlled and managed.
 
-To ensure the neccessary page is available and working simply navigate to
-`/content/asset-selectrors/ms-office.html' in your web browser.
+To ensure the necessary page is available and working, simply navigate to
+`/content/asset-selectrors/ms-office.html` in your web browser.
 
 ### Office Add-in Configuration
 
 The primary step necessary for the add-in to work is to update the manifest
-files to direct users to your AEM instance. Two [manifest files](#sample_files)
-are available as part of this package. Simply download these files and edit them
-to point to the correct URL. There are two lines which will need to be editted:
-
-+ Line 19: SourceLocation
-+ Line 71: Taskpane.Url
-
-Update the URL in the DefaultValue on these two lines to the AEM Sites instance to be used.
+files to direct users to your AEM instance.
+Download the above Cloud manifest.xml and edit them to point to the correct URL.
+
+Follow the Steps below:
+
+1. Update `manifest.xml` with your deployed AEM instance url. Replace `p00000-e000000` with your Cloud Program IDs.
+    1. IconUrl
+    2. HighResolutionIconUrl
+    3. SupportUrl
+    4. AppDomain
+    5. SourceLocation
+    6. bt:Images
+    7. bt:Urls
+2. Adjust Strings in `manifest.xml` to your liking
+    1. DisplayName
+    2. Description
+    3. GetStarted.Title
+    4. CommandsGroup.Label
+    5. TaskpaneButton.Label
+    6. GetStarted.Description
+    7. TaskpaneButton.Tooltip
+3. Update `<Id>` in `manifest.xml` to a new unique id
+    - e.g `<Id>gy4of8YpaRzkTqx3N75m</Id>`
+4. Include and Deploy ACS Commons in your project
+    - or install the provided package for just the asset selector
+        - <https://share.lucanerlich.com/s/FXB9r6SwWJPHxxe>
```

Comment:
is it possible to move that elsewhere to make it independent from your system? Maybe even uploading it into github?

Reply:
Yes. Wip link. Should be uploaded to either the repo or somewhere else "adobe".